### ISSUE TITLE
0.4.1 ClientMoved 'clid' is a "ChannelId" 

### ISSUE THREAD

**Issue by Bwahharharrr** [MrGunflame/ts3-rs#5]
https://github.com/MrGunflame/ts3-rs/blob/a7fceaad08032b91a8372da9759f00b9ea35db39/ts3/src/event.rs#L425

```rust
pub struct ClientMoved {
    pub ctid: ChannelId,
    pub reasonid: ReasonId,
    pub invokerid: ClientId,
    pub invokername: String,
    pub invokeruid: String,
    pub clid: ChannelId,
}
```

the `clid` is labeled as a `ChannelId` which seems odd, since `cid` is normally the `ClientId`.
This I believe is what is leading to me getting an `invokerid` of 0, and a `clid` actually being the client id value for the move event

See screenshot
![https://i.imgur.com/8Ga8GDJ.png](https://i.imgur.com/8Ga8GDJ.png)

**Comment by MrGunflame** [comment]
Yeah, that doesn't look correct. A fix will have to wait until `0.5.0` to prevent breakage. For now I'd just recommend to extract the `u64` from the struct.
